src/Params/EnvelopeParams.cpp

```diff
@@ -263,9 +291,9 @@ EnvelopeParams::EnvelopeParams(unsigned char Penvstretch_,
                                const AbsTime *time_):
         time(time_), last_update_timestamp(0)
 {
-    A_dt  = 0.009;
+    A_dt  = 0.0001f;
     D_dt  = 0.009;
-    R_dt  = 0.009;
+    R_dt  = 0.499f;
```

Comment:
IIRC these values are overwritten later, so is the change in the release needed?

Reply:
fixed